package.json

```diff
@@ -20,6 +20,7 @@
         "cypress": "9.3.1",
         "http-server": "14.1.0",
         "lerna": "4.0.0",
+        "maildev": "github:padloc/maildev",
```

Comment:
Switched to a custom for of maildev since the original had some dependencies with high and critical vulnerabilities. See https://github.com/padloc/maildev/commit/a3ac6bf682dfb7cbb08b8b86e7b8afc9e5e41c66